rest/src/main/kotlin/NamedFile.kt

```diff
@@ -1,12 +1,12 @@
 package dev.kord.rest
 
-import java.io.InputStream
+import io.ktor.utils.io.*
 
-public class NamedFile(public val name: String, public val inputStream: InputStream) {
+public class NamedFile(public val name: String, public val readChannel: ByteReadChannel) {
```

Comment:
oh yeah, my bad!

Reply:
I'm unsure what to do with `NamedFile`. Just provide a secondary constructor that accepts an `InputStream`? Then it wouldn't be fully binary compatible, would it? Since `inputStream` still wouldn't be a member and `component2` also does not return the same thing as before.  
Sorry, I'm not well versed in maintaining binary compat in Kotlin libraries.

Reply:
you'll need:
- secondary constructor
- `inputStream` property with getter converting the `ByteReadChannel` to `InpuStream`
- hidden component2:
```kotlin
@Deprecated("Binary compatibility", level = HIDDEN)
@JvmName("component2")
public fun _component2(): InputStream = inputStream
```

Reply:
ooh, that's what HIDDEN is for, thanks!